data/human/transport missions.txt

```diff
@@ -3632,3 +3632,123 @@ mission "Antibiotics To South"
 		payment
 		payment 20000
 		dialog phrase "generic cargo delivery payment"
+
+
+mission "Pirate Duel"
+	name "Fight at Umbral"
+	description `Head as quickly as you can to the Umbral system, where you will duel with a small militia vessel named the "Esca". After fighting, go to <destination> to follow up.`
+	minor
+	deadline 5 1
+	source
+		near "Tarazed" 1 5
+		government "Republic" "Free Worlds"
+	waypoint "Umbral"
+	destination "Wayfarer"
+	
+	to offer
+		"combat rating" > 20
+		"combat rating" < 400
+		random < 35
+		not "ships: Medium Warship"
+		not "ships: Heavy Warship"
+		not "ships: Heavy Freighter"
+		or
+			"ships: Light Warship" == 1
+			"ships: Light Freighter" == 1
+			"ships: Transport" == 1
+			"ships: Interceptor" == 1
+	
+	on offer
+		conversation
+			`While you are walking around the spaceport, you accidentally bump into a sturdily-built man. "Oh, I'm sorry," he responds in a gruff voice. When they take a closer look at you, his expression changes to shock. "Oh! You're Captain <last>!"`
+			choice
+				`	"How did you know my name?"`
+					goto offer
+				`	"I don't know who you're talking about."`
+			`	The man's expression changes to one of slight dissapointment. "Oh. I see." They walk away without another word.`
```

Comment:
Same here: "they" to a "he", then makes "walk" "walks", since you're now using a singular pronoun.